app/src/main/res/layout/item_rolled_dice.xml

```diff
@@ -6,53 +6,53 @@
     android:layout_height="wrap_content"
     android:paddingVertical="4dp">
 
-
     <ImageView
-        android:id="@+id/rolledDiceIcon"
-        android:layout_width="64dp"
-        android:layout_height="64dp"
+        android:id="@+id/rolled_dice_icon"
```

Comment:
You were very inconsistent with IDs in your layouts. Stick to snake_case in XML. Binding converts it to camelCase for you. Being consistent throughout is important. It shows a sense of attention to detail.